sql/item_mods.sql

```diff
@@ -52119,6 +52119,18 @@ INSERT INTO `item_mods` VALUES (23795,1,1); -- DEF: 1
 -- Ziamet Nails
 INSERT INTO `item_mods` VALUES (23796,1,1); -- DEF: 1
 
+-- Poroggo Cass. +1
+INSERT INTO `item_mods` VALUES (23804,15,20);   -- FIRE_RES: 20
+INSERT INTO `item_mods` VALUES (23804,16,20);   -- ICE_RES: 20
+INSERT INTO `item_mods` VALUES (23804,17,20);   -- WIND_RES: 20
+INSERT INTO `item_mods` VALUES (23804,18,20);   -- EARTH_RES: 20
+INSERT INTO `item_mods` VALUES (23804,19,20);   -- THUNDER_RES: 20
+INSERT INTO `item_mods` VALUES (23804,20,20);   -- WATER_RES: 20
+INSERT INTO `item_mods` VALUES (23804,21,20);   -- LIGHT_RES: 20
+INSERT INTO `item_mods` VALUES (23804,22,20);   -- DARK_RES: 20
+INSERT INTO `item_mods` VALUES (23804,416,1000);   -- occasionaly nullify physical dmg
+INSERT INTO `item_mods` VALUES (23804,476,1000);   -- occasionaly nullify magic dmg
+
```

Comment:
Is there a "occasionaly nullify breath dmg"? couse that would fall here aswell.
This doesnt really convince me though. "Damage taken" is a mod in itself, that encompases ALL damage types.

Also, this comments need to be converted to the actual mod name.

Reply:
There is not I changed the mods to "element null" but does not work on breath. Any other suggestions?

Reply:
@Xaver-DaRed i think the null element mods should work vs any source of that element, breath included. we should start an issue to confirm and call for any needed updates to match what retail does.